korman/properties/modifiers/gui.py

```diff
@@ -427,6 +427,14 @@ class PlasmaLinkingBookModifier(PlasmaModifierProperties, PlasmaModifierLogicWiz
                                 description="The object the avatar will click on to activate the Linking Book GUI",
                                 type=bpy.types.Object,
                                 poll=idprops.poll_mesh_objects)
+    shareable = BoolProperty(name="Shareable",
+                             description="Enable the Book to be Shareable (MOUL private instance only)",
+                             default=True,
+                             options=set())
+    share_region = PointerProperty(name="Share Region (optional)",
+                                       description="Sets an alternate share region (optional)",
+                                       type=bpy.types.Object,
+                                       poll=idprops.poll_mesh_objects)
```

Comment:
Looks like the indentation needs to be fixed here.

Reply:
That ugly whitespace! It's gone now.